finetune/target_models/sequence_labeling.py

```diff
@@ -314,6 +346,17 @@ def _predict(self, zipped_data, per_token=False, **kwargs):
                     doc_annotations,
                 )
             ]
+        elif return_negative_probs:
+            classes = self.input_pipeline.label_encoder.classes_
+            output = []
+            for anno, probas in zip(doc_annotations, all_doc_level_probas):
+                output.append(
+                    {
+                        "prediction": anno,
+                        "negative_probs": dict(zip(classes, probs))
```

Comment:
I'd use "proba" to keep the naming consistent with predict_proba

Reply:
Or maybe better yet, "negative_confidence" since we've set a precedent w/ "confidence"